testhelpers/org.eclipse.n4js.ui.tests.helper/src/org/eclipse/n4js/tests/builder/AbstractBuilderTest.java

```diff
@@ -59,15 +58,10 @@
 /**
  * @author Sven Efftinge - Initial contribution and API
  */
-@SuppressWarnings("restriction")
 @RunWith(XtextRunner.class)
 @InjectWith(N4JSUiInjectorProvider.class)
 public abstract class AbstractBuilderTest {
 
-	static {
-		EclipseGracefulUIShutdownEnabler.enableOnce();
-	}
```

Comment:
Where to / Why did you remove the above?

Reply:
duplicate, there is one below: https://github.com/eclipse/n4js/pull/930/files/69acff612f1094f6118f9825a68c9edbe2700cf3#diff-f9fa75a8643c1a4ad55c64b462cce924R69